ERCS/erc-7567.md

```diff
@@ -0,0 +1,155 @@
+---
+eip: 7567
+title: Splitable utility NFT extension for ERC-721
+description: Interfaces for adding splitable utility properties to ERC-721
+author: Aditya Shukla (@adityashukla700), Akshat Modani (@Akshat12210), Namish Khandelwal (@Namishk), Sparsh Goud (@sparsh76)
+discussions-to: https://ethereum-magicians.org/t/splitable-utility-nft-token-for-erc-721/16380
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-11-28
+requires: 165, 721
+---
+  
+## Abstract
+
+  
+
+This standard is an extension of [ERC-721](./eip-721.md). It proposes splitable utility properties to a NFT, which will allow user to have or trade specific utilities instead of whole set as an NFT. It gives creators ability to add new utility functionality to their tokens.
+
+
+## Specification
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+**Overview**
+
+Multiutility token contracts, compliant with the [ERC-721](./eip-721.md) standard, `MAY` implement this extension to provide standard methods for `managing` the `distribution` and `transfer` of `utilities` associated with NFTs.
```

Comment:
More specifically, only use backticks (\`) for code snippets. Use italics or bold for non-code terms with specific definitions.